src/test/scala/org/broadinstitute/dsde/agora/server/webservice/AgoraImportSpec.scala

```diff
@@ -19,49 +19,33 @@ class AgoraImportSpec extends ApiServiceSpec {
   override def beforeAll() = {
     ensureDatabasesAreRunning()
     testEntityTaskWcWithId = patiently(agoraBusiness.insert(testEntityTaskWc, mockAuthenticatedOwner.get))
-    patiently(agoraBusiness.insert(testEntityWorkflowWithExistentWdlImport, mockAuthenticatedOwner.get))
+    //patiently(agoraBusiness.insert(testEntityWorkflowWithExistentWdlImport, mockAuthenticatedOwner.get)) GAWB-937 means no more imports allowed
   }
 
   override def afterAll() = {
     clearDatabases()
   }
 
-  "MethodsService" should "return a 201 created when posting a WDL with an invalid import statement" in {
+  "MethodsService" should "return a 400 when posting a WDL with an invalid import statement" in {
     Post(ApiUtil.Methods.withLeadingVersion, testBadAgoraEntityInvalidWdlImportFormat) ~>
       methodsService.postRoute ~> check {
-      assert(status == Created)
+      assert(status == BadRequest)
 //      assert(responseAs[String] != null)
     }
   }
 
-  "MethodsService" should "return a 201 created when posting a WDL with an import statement that references a non-existent method" in {
+  "MethodsService" should "return a 400 when posting a WDL with an import statement that references a non-existent method" in {
     Post(ApiUtil.Methods.withLeadingVersion, testBadAgoraEntityNonExistentWdlImportFormat) ~>
       methodsService.postRoute ~> check {
-      assert(status == Created)
+      assert(status == BadRequest)
 //      assert(responseAs[String] != null)
     }
```

Comment:
Is there a reason we are leaving some of the code commented out rather than just deleting it?